keyboards/acheron/shark/config.h

```diff
@@ -0,0 +1,261 @@
+/*
+Copyright 2019 Alvaro Volpato and Steve
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
You might want to tailor these various USB descriptor values to better represent your board.

Reply:
Well, `FEED` is the default for most of the boards.  Generaly, it's the product ID that would be changed, over the vender ID.

Reply:
@zvecr what do you suggest? As @drashna said, I did use the default

Reply:
I just marked VENDOR_ID as the start of the block, but generally yes, PRODUCT_ID would be the bare minimum to change.

Reply:
What would be a suitable PRODUCT_ID? None I find in the lists are

Reply:
Honestly, whatever you want to use.

Reply:
PRODUCT_ID updated in commit [2bd5481](https://github.com/qmk/qmk_firmware/pull/7090/commits/2bd548174c0a672a94d4f7737430e8f72d51a0be).

Thanks guys!